lib/stoplight/data_store/memory.rb

```diff
@@ -1,19 +1,19 @@
 # frozen_string_literal: true
 
-require 'monitor'
+require "monitor"
 
 module Stoplight
   module DataStore
     # @see Base
     class Memory < Base
       include MonitorMixin
-      KEY_SEPARATOR = ':'
+      KEY_SEPARATOR = ":"
 
       def initialize
         @failures = Hash.new { |h, k| h[k] = [] }
         @states = Hash.new { |h, k| h[k] = State::UNLOCKED }
         @last_notifications = {}
-        super() # MonitorMixin
```

Comment:
Is this a linter fix, or do we no longer want to depend on the parent method arguments?

Reply:
Yep this is `bundle exec standardrb --fix` 

> Style/SuperArguments: Call `super` without arguments and parentheses when the signature is identical.

As any opinionated tool, it may have opinions that we disagree with, but in general, I find its default settings more sensible.

Reply:
Oh, I thought that method signatures were not identical. I'm good with this change then.